pl/python_introduction/README.md

```diff
@@ -773,7 +773,7 @@ else:
     print('Hej anonimie!')
 ```
 
-i uruchommy go:
+i uruchomimy go:
```

Comment:
Might be 'uruchommy go', as in other parts

Reply:
That's exactly how it looks on Crowdin, so if You want to fix it, do it there. :)

Reply:
Done.